_tests/browser_tests/jenkins_conf.js

```diff
@@ -0,0 +1,40 @@
+var environment = require('./environment.js');
```

Comment:
There are about 40 new warnings when linting this file due to missing spaces inside parens.